tsconfig.json

```diff
@@ -1,8 +1,14 @@
 {
   "extends": "./tsconfig.build.json",
-  "include": ["src/**/*.ts", "test/**/*.ts"],
-  "exclude": ["dist"],
+  "include": [
+    "src/**/*.ts",
+    "test/**/*.ts"
+  ],
+  "exclude": [
+    "dist"
+  ],
   "compilerOptions": {
-    "rootDir": "."
+    "rootDir": ".",
+    "esModuleInterop": false,
```

Comment:
Why turn this back off here?

Reply:
Not the original author, but I think this is to fix the issue I filed: https://github.com/getsentry/sentry-electron/issues/431

I can't add sentry to my compiled application (that uses ES modules) due to `Cannot find module "deepmerge"` issue in the compiled version of `/esm/common/merge.js`